src/coprocessor/endpoint.rs

```diff
@@ -600,6 +600,8 @@ impl<E: Engine> Endpoint<E> {
                                     response.set_locked(lock_info);
                                 }
                                 response.set_other_error(resp.take_other_error());
+                                // keep the exec details already generated.
+                                response.set_exec_details_v2(resp.take_exec_details_v2());
```

Comment:
Maybe we could combine the `execution path tracker` together with the `cop task tracker` and set execution details all from one tracker later.